js/components/Chart.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
This component is just a stub for now. Implementation will be included in my next PR.